dotnet/samples/README.md

```diff
@@ -0,0 +1,10 @@
+## Kernel Samples
+
+| Type                                                                        | Description                                                                                              |
+| --------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------- |
+| [`Getting started`](./GettingStarted/README.md)                             | A single step-by-step tutorial to get started                                                            |
+| [`Tutorials`](./Tutorials/README.md)                                        | More in depth step-by-step tutorials                                                                     |
```

Comment:
How about some thing: `Here you will find more in depth tutorials on topics such as, ...`

I think we should list what will be included in this section

Reply:
Removed until we have our first in depth tutorial.